Sources/SnowManLogic/ViewLogic/MockView.swift

```diff
@@ -0,0 +1,40 @@
+public class MockView: Viewable {
+	var ui: Receivable 
+	var guess = ""
+
+	public init(ui: Receivable) {
+		self.ui = ui
+	}
+
+	public func receiveGuess() -> String {
+		guess = ui.getUserInput()
+		if (containsOnlyLetters(guess)) {
+			return guess
+		} else {
+			return "Invalid"
+		}
+	}
+
+	public func assignBlanks(gameWord: String, correctGuesses: [String]) -> String {
+		var blanks: [String] = []
```

Comment:
I fixed this. The only difference in the Swift docs between the two options is that `something = []` is more typical when a type has already been established on the variable. I assume this is just a style thing.